src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx

```diff
@@ -282,56 +332,85 @@ class ReleaseHealthRequest extends React.Component<Props, State> {
       ?.reduce((acc, group) => acc + group.totals[field], 0);
   };
 
-  getTimeSeries = (
-    version: string,
-    project: number,
-    display: DisplayOption,
-    healthStatsPeriod?: string
-  ) => {
-    if (healthStatsPeriod === 'auto') {
+  getTimeSeries = (version: string, project: number, display: DisplayOption) => {
+    const {healthStatsPeriod} = this.props;
+    if (healthStatsPeriod === HealthStatsPeriodOption.AUTO) {
       return this.getPeriodTimeSeries(version, project, display);
     }
 
     return this.get24hTimeSeries(version, project, display);
   };
 
   get24hTimeSeries = (version: string, project: number, display: DisplayOption) => {
-    const {totalCountByReleaseIn24h} = this.state;
+    const {totalCountByReleaseIn24h, totalCountByProjectIn24h} = this.state;
     const field = this.displayToField(display);
 
+    const intervals = totalCountByProjectIn24h?.intervals ?? [];
+
+    const projectData = totalCountByProjectIn24h?.groups.find(
+      ({by}) => by.project === project
+    )?.series[field];
+
+    const releaseData = totalCountByReleaseIn24h?.groups.find(
+      ({by}) => by.project === project && by.release === version
+    )?.series[field];
+
     return [
+      {
+        seriesName: t('Total Project'),
+        data: intervals?.map((interval, index) => ({
+          name: moment(interval).valueOf(),
+          value: projectData?.[index] ?? 0,
+        })),
+      },
       {
         seriesName: t('This Release'),
-        data:
-          totalCountByReleaseIn24h?.groups
-            .find(({by}) => by.project === project && by.release === version)
-            ?.series[field]?.map((value, index) => ({
-              name: moment(totalCountByReleaseIn24h.intervals[index]).valueOf(),
-              value,
-            })) ?? [],
+        data: intervals?.map((interval, index) => ({
+          name: moment(interval).valueOf(),
+          value: releaseData?.[index] ?? 0,
+        })),
       },
     ];
   };
 
   getPeriodTimeSeries = (version: string, project: number, display: DisplayOption) => {
-    const {statusCountByReleaseInPeriod} = this.state;
+    const {statusCountByReleaseInPeriod, statusCountByProjectInPeriod} = this.state;
     const field = this.displayToField(display);
+
+    const intervals = statusCountByProjectInPeriod?.intervals ?? [];
+
+    const projectData = statusCountByProjectInPeriod?.groups
+      .filter(({by}) => by.project === project)
+      ?.reduce((acc, group) => {
+        group.series[field].forEach(
+          (value, index) => (acc[index] = (acc[index] ?? 0) + value)
```

Comment:
Is it useful to have a local variable with this function as it shows up a few times here.

Reply:
yes ❤️